CHANGELOG.md

```diff
@@ -21,6 +21,9 @@ This is a changelog for Piwik platform developers. All changes for our HTTP API'
 ### Internal Change
 * The option `branch` of the console command `development:sync-system-test-processed` was removed as it is no longer needed.
 
+### APIs
+* There is a new event `CronArchive.getIdSitesNotUsingTracker` that allows you to set a list of idSites that do not use the Tracker API to make sure we archive these sites if needed.
```

Comment:
I had it something like `getIdSitesToForceArchivingFor ` (before I squashed them) but found it's not the right name as forcing might be misleading etc. Eg when someone is using `skip-sites` or `force-sites` this site might be not archived and therefore not forced. Also it might be not forced if there are no visits see your link. I found the word `force` too strong. I think from a 3rd party developer point of view it's maybe better understandable to have it `getIdSitesNotUsingTracker`. It only requires knowing whether a sites is using the Tracker or not. Maybe `getIdSitesNotUsingTrackerAPI` would be clearer? 

I added it to make it work for MultiSites but haven't tested it for any actual 3rd data that only uses it's own plugin archiver (like MobileAppStore maybe does?). Do you think it's easy to fix? One problem that I'm not sure about is that it looks like we're working with multiple idSites there and not just one site see https://github.com/piwik/piwik/blob/master/core/ArchiveProcessor/Loader.php#L133 `$this->params->getIdSites()` . Probably it's always only one site though. I reckon that's what you suggested and I now understand what you mean with it clashes with MetaSites. It does sound like a new event would be needed but that's kinda too complex. Would it be something like `getIdSitesNotUsingLogTables`? It's maybe a bit too technical as many developers won't know the log tables.

* MetaSites is not using Tracker API but Log tables
* MobileAppStore is not using Tracker API and not Log tables (I think)

Do you have an idea for that event name? I could maybe implement it